### DISCUSSION TITLE
svelte 4.0.x

### DISCUSSION THREAD

**Discussion by diramazioni** [f5/unovis#250] [Q&A]
I'm trying to integrate unovis into my svelte 4 project but the simple quickstart example doesn't seems to produce the expected lines. 
Then I noticed that the running example in stackblitz all use svelte 3.5 so I wonder if is a versioning issue

```
<script lang="ts">
  import { VisXYContainer, VisLine, VisAxis } from '@unovis/svelte'


  type DataRecord = { x: number, y: number }
  export let data: DataRecord[] = [
    { x: 0, y: 0 },
    { x: 1, y: 2 },
    { x: 2, y: 1 },
  ]
  const x = (d: DataRecord) => d.x
  const y = (d: DataRecord) => d.y

</script>


<VisXYContainer data={data} height={300}>
  <VisLine {x} {y}/>
  <VisAxis type="x"/>
  <VisAxis type="y"/>
</VisXYContainer>
```

**Comment by rokotyan** [comment]
Hi @diramazioni

Happy to take a look, will you be able to share an example via CodeSandbox or StackBlitz?

**Comment by SenneVanderAuwera** [comment]
I'm having the same issue. Here is a link to a stackblitz example: https://stackblitz.com/edit/sveltejs-kit-template-default-bbabwa?file=src%2Froutes%2F%2Blayout.svelte

**Comment by rokotyan** [comment]
Thanks @SenneVanderAuwera!

Confirming that there's a problem. We'll take a look

(cc @reb-dev)
